ml-algorithms/src/main/java/org/opensearch/ml/engine/indices/MLIndex.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.indices;
+
+import static org.opensearch.ml.common.CommonValue.ML_AGENT_INDEX;
+import static org.opensearch.ml.common.CommonValue.ML_AGENT_INDEX_MAPPING;
+import static org.opensearch.ml.common.CommonValue.ML_AGENT_INDEX_SCHEMA_VERSION;
+import static org.opensearch.ml.common.CommonValue.ML_MEMORY_MESSAGE_INDEX;
+import static org.opensearch.ml.common.CommonValue.ML_MEMORY_MESSAGE_INDEX_MAPPING;
+import static org.opensearch.ml.common.CommonValue.ML_MEMORY_MESSAGE_INDEX_SCHEMA_VERSION;
+import static org.opensearch.ml.common.CommonValue.ML_MEMORY_META_INDEX;
+import static org.opensearch.ml.common.CommonValue.ML_MEMORY_META_INDEX_MAPPING;
+import static org.opensearch.ml.common.CommonValue.ML_MEMORY_META_INDEX_SCHEMA_VERSION;
+
+public enum MLIndex {
```

Comment:
Why not include other existing index like model group index ?
https://github.com/opensearch-project/ml-commons/blob/feature/agent_framework_dev/ml-algorithms/src/main/java/org/opensearch/ml/engine/indices/MLIndex.java

Reply:
moved all indices here